plane/src/typed_unix_socket/mod.rs

```diff
@@ -1,30 +1,17 @@
 use chrono::Duration;
 use serde::{Deserialize, Serialize};
-use uuid::Uuid;
 
 use crate::util::ExponentialBackoff;
 
 pub mod client;
 pub mod server;
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct IDedMessage<T> {
-    id: Uuid,
+pub struct WrappedMessage<T> {
+    id: Option<String>,
     message: T,
 }
```

Comment:
The semantics here are: if ID is given, this is a request/response; otherwise this is an event.

This replaces both `IDedMessage` and `Wrapped{Client,Server}MessageType`.

Reply:
> The semantics here are: if ID is given, this is a request/response; otherwise this is an event.

Nit: can we add a comment about this?